src/components/CurrencyTable/index.js

```diff
@@ -1,3 +1,34 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { makeStyles } from '@material-ui/core/styles';
+import CircularProgress from '@material-ui/core/CircularProgress';
 import CurrencyTable from './CurrencyTable';
+import Text from '../Text';
 
-export default CurrencyTable;
+const useStyles = makeStyles(() => ({
+  center: {
+    display: 'flex',
+    justifyContent: 'center'
+  }
+}));
+
+const CurrencyTableWrapper = () => {
+  const stockState = useSelector((state) => state.stock);
```

Comment:
How about to destructure that object in here to avoid writing long stockState.isLoading + isError?